src/components/todo/TodoInput.js

```diff
@@ -1,11 +1,38 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import styled from 'styled-components';
 
+const InputWrapper = styled.div`
+    display:flex;
+    justify-content:center;
+    border-radius: 25px;
+    padding:50px;
+    background-color:#dcffa8;
+`
+
+const AddButton = styled.input`
+    background-color:Green;
+    color:White;
+`
+
 // 힌트 : Row, Input, Button
 class TodoInput extends Component {
   render() {
-    return <div>화이팅^^</div>;
+    return (
+      <InputWrapper>
+        <form action="/create_process" method="post"
+          onSubmit={function (e) {
+            e.preventDefault(); //form 제출했을 때 redirect 방지
+            this.props.onSubmit( //submit 버튼 눌렀을 때 content라는 이름의 input에 입력한 내용 전송
+              e.target.content.value
+            );
+            e.target.content.value = null; //다시 새로운 내용 입력할 수 있도록 null 대입
+          }.bind(this)}>
+          <input type="text" name="content" placeholder="TODO를 입력하세요."></input>
```

Comment:
```suggestion
          <input type="text" value={this.state.content} placeholder="TODO를 입력하세요."></input>
```
변화하는 값에 대해서 최대한 state를 활용해주시면 확장성이 매우 좋아집니다!
예를 들어, 입력값이 10글자 이하인 경우 입력창 밑에 '10글자 이상 입력해주세요'라는 글씨를 띄우고 싶은 상황에서 유용하게 쓸 수 있죠.